po/it.po

```diff
@@ -1872,68 +1863,68 @@ msgstr ""
 
 #: rust/libnewsboat/src/configpaths.rs:146
 msgid "Migrating configs and data from Newsbeuter's XDG dirs..."
-msgstr ""
+msgstr "Migrando configurazioni e dati da cartelle XDG di Newsbeuter..."
 
 #: rust/libnewsboat/src/configpaths.rs:202
 msgid "Migrating configs and data from ~/.newsbeuter/..."
-msgstr ""
+msgstr "Migrando configurazioni e dati da ~/.newsbeuter/..."
 
 #: rust/libnewsboat/src/configpaths.rs:214
-#, fuzzy
 msgid "Aborting migration because mkdir on `%s' failed: %s"
-msgstr "Errore: apertura del file cache `%s' fallita: %s"
+msgstr "Migrazione interrotta per fallimento di mkdir su `%s': %s"
 
 #: rust/libnewsboat/src/filterparser.rs:63
-#, fuzzy
 msgid "attribute name"
-msgstr "attributo non trovato"
+msgstr "nome attributo"
 
 #: rust/libnewsboat/src/filterparser.rs:64
+#, fuzzy
 msgid "one of: =~, ==, =, !~, !=, <=, >=, <, >, between, #, !#"
-msgstr ""
+msgstr "uno tra: =~, ==, =, !~, !=, <=, >=, <, >, between, #, !#"
```

Comment:
I'm guessing you marked those as "fuzzy" because of "between". You did everything right here: `between` is an operator, so it shouldn't be translated. We already made a note to ourselves to make it clearer to the translators #1354.

Reply:
Yes, exactly, I wasn't quite sure about it, removing it then!